portfolio/src/main/webapp/index.html

```diff
@@ -22,10 +30,30 @@ <h1>Mariah Crawford's Portfolio</h1>
       <p>Click here to get a random fun fact about Mariah:</p>
       <button onclick="addRandomFunFact()">Generate Fun Fact</button>
       <div id="fun-fact-container"></div>
-      
-      <p>View Comments Below:</p>
-      <ul id="messages-container"></ul>
+        
+        <button onclick="fetchBlobstoreUrlAndShowForm()">Write a comment</button>*/
```

Comment:
I see that you removed the "*/" from the end of this line in next PR. So we can ignore it here.

Reply:
Sounds good